kuka_rsi_hw_interface/include/kuka_rsi_hw_interface/udp_server.h

```diff
@@ -44,6 +44,18 @@
 #include <string>
 #include <stdexcept>
 
+#ifdef WIN32
+#include <windows.h>
+#undef ERROR
+#undef OK
```

Comment:
A bit late, but would using what @seanyen did in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1023 perhaps be possible here (and cleaner)?

Reply:
Oh yea, that is a better fix. I'll update it, thanks!